engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/CreateDeploymentMultiplePartitionsTest.java

```diff
@@ -43,13 +42,13 @@ public final class CreateDeploymentMultiplePartitionsTest {
   public static final String PROCESS_ID = "process";
   public static final int PARTITION_ID = DEPLOYMENT_PARTITION;
   public static final int PARTITION_COUNT = 3;
-  @ClassRule public static final EngineRule ENGINE = EngineRule.multiplePartition(PARTITION_COUNT);
   private static final BpmnModelInstance PROCESS =
       Bpmn.createExecutableProcess(PROCESS_ID).startEvent().endEvent().done();
   private static final BpmnModelInstance PROCESS_2 =
       Bpmn.createExecutableProcess("process2").startEvent().endEvent().done();
   private static final String DMN_DECISION_TABLE = "/dmn/decision-table.dmn";
   private static final String DMN_DECISION_TABLE_V2 = "/dmn/decision-table_v2.dmn";
+  @Rule public final EngineRule engine = EngineRule.multiplePartition(PARTITION_COUNT);
```

Comment:
❓ Why is this necessary? 

🔧 This slows down these tests significantly. Let's find a solution to reuse the engine between the different cases if possible.